src/generate/helpers/reduxBindings/writeInitializer.ts

```diff
@@ -0,0 +1,50 @@
+import { camelize, pascalize } from '@rvoh/dream'
+import * as fs from 'node:fs/promises'
+import * as path from 'node:path'
+import psychicPath from '../../../helpers/path/psychicPath.js'
+
+export default async function writeInitializer({ exportName }: { exportName: string }) {
+  const pascalized = pascalize(exportName)
+  const camelized = camelize(exportName)
+
+  const destDir = path.join(psychicPath('conf'), 'initializers', 'openapi')
+  const initializerFilename = `${camelized}.ts`
+  const initializerPath = path.join(destDir, initializerFilename)
+
+  try {
+    await fs.access(initializerPath)
+    return // early return if the file already exists
+  } catch {
+    // noop
+  }
+
+  try {
+    await fs.access(destDir)
+  } catch {
+    await fs.mkdir(destDir, { recursive: true })
+  }
+
+  const contents = `\
+import { DreamCLI } from '@rvoh/dream'
+import { PsychicApp } from '@rvoh/psychic'
+import AppEnv from '../AppEnv.js'
+
+export default function initialize${pascalized}(psy: PsychicApp) {
```

Comment:
When we formalize the initializer generator pattern for future plugin authors, we might want to provide an API that includes all the standard stuff so that initializers come out in a standard way.